index.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+var bodyParser = require('body-parser');
+var express = require('express');
+var gitHubWebHook = require('express-github-webhook');
+var nconf = require('nconf');
+var Promise = require('bluebird');
+
+var GitHubServer = require('./lib/GitHubServer');
+var getGoogleGroupLinks = require('./lib/getGoogleGroupLinks');
+
+var app = express();
+module.exports = app;
+
+nconf.env('__')
+    .file({
+        file: 'config.json'
+    });
+
+nconf.defaults({
+    port: 5000,
+    secret: '', // Repository secret to verify incoming WebHook requests from GitHub
+    gitHubToken: '', // Token used to verify outgoing requests to GitHub repository
+    repository: '', // Repository to scan for outdated pull requests and bump them
+    listenPath: '/' // Path on which to listen for incoming requests
```

Comment:
For values without sensible defaults, we should actually make sure they exist and exit the application with an error if they don't.